packages/ckeditor5-dev-changelog/src/utils/logchangelogfiles.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import type { SectionsWithEntries } from '../types.js';
+import chalk from 'chalk';
+import { logInfo } from './loginfo.js';
+
+/**
+ * Logs information about changelog sections and their entries to the console.
+ * This function provides a summary of changes that will be included in the changelog.
+ */
+export function logChangelogFiles( sections: SectionsWithEntries ): void {
+	// todo in the followup: Display invalid records differently. Add filepath, and display what is invalid:
+	//  - type, scope, breaking change, closes or see.
+
+	logInfo( `📍 ${ chalk.cyan( 'Listing the changes...' ) }\n` );
```

Comment:
For unknown reasons (probably lack of support in the used font) it may happen that emoji is not rendered correctly in the terminal:

```
� Listing the changes...
```

Would you like to consider using some more basic Unicode characters that work in all environments? Example:

```
○ Listing the changes...
```

Reply:
This is insane:

![obraz](https://github.com/user-attachments/assets/87a5243e-e1b9-4698-9eeb-62c89c04e753)

It should work in my environment. No idea why it prints `�` 🤔

Reply:
Yeah, I took this emoji from the previous changelog tool, but I'll change it to the ○ char, so that it's supported on all systems.